<?xml version="1.0" encoding="UTF-8"?>
<testrun name="DocumentFrameTest" project="JWP" tests="11" started="11" failures="1" errors="0" ignored="0">
  <testsuite name="jmonkey.office.jwp.DocumentFrameTest" time="0.994">
    <testcase name="testConstruction" classname="jmonkey.office.jwp.DocumentFrameTest" time="0.947"/>
    <testcase name="test_getName" classname="jmonkey.office.jwp.DocumentFrameTest" time="0.002"/>
    <testcase name="test_internalFrameOpened" classname="jmonkey.office.jwp.DocumentFrameTest" time="0.001"/>
    <testcase name="test_internalFrameClosing" classname="jmonkey.office.jwp.DocumentFrameTest" time="0.002"/>
    <testcase name="test_internalFrameClosed" classname="jmonkey.office.jwp.DocumentFrameTest" time="0.001"/>
    <testcase name="test_internalFrameIconified" classname="jmonkey.office.jwp.DocumentFrameTest" time="0.001"/>
    <testcase name="test_internalFrameDeiconified" classname="jmonkey.office.jwp.DocumentFrameTest" time="0.002"/>
    <testcase name="test_internalFrameDeactivated" classname="jmonkey.office.jwp.DocumentFrameTest" time="0.001"/>
    <testcase name="test_focusLost" classname="jmonkey.office.jwp.DocumentFrameTest" time="0.001"/>
    <testcase name="test_activate" classname="jmonkey.office.jwp.DocumentFrameTest" time="0.013"/>
    <testcase name="test_focus_gained" classname="jmonkey.office.jwp.DocumentFrameTest" time="0.022">
      <failure>java.lang.AssertionError: unknown exception found null source
	at org.junit.Assert.fail(Assert.java:91)
	at jmonkey.office.jwp.DocumentFrameTest.test_focus_gained(DocumentFrameTest.java:122)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.runners.BlockJUnit4ClassRunner.runNotIgnored(BlockJUnit4ClassRunner.java:79)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:71)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:49)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:193)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:52)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:191)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:42)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:184)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:236)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)

</failure>
    </testcase>
  </testsuite>
</testrun>
